config/manager/kustomization.yaml

```diff
@@ -12,5 +13,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/fgiloux/settings-operator
+  newName: quay.io/redhat-pipeline-service/settings-operator
```

Comment:
This registry address is where we would store settings-operator officially, right? I would like to refrain from pushing my images unintentionally here. Shouldn't we keep this address to something generic like `quay.io/user/settings-operator`? It would be overridden when `IMG` in `make` command is passed.